src/js/components/TextInput/TextInput.js

```diff
@@ -152,18 +153,39 @@ const TextInput = forwardRef(
     }, [onSuggestionsOpen, suggestions]);
     /* eslint-enable react-hooks/exhaustive-deps */
 
+    // choose the best suggestion, either the explicit default or the one
+    // that matches the current value
+    const bestSuggestion = useCallback(() => {
```

Comment:
After I select a value, if I place focus in the input and press "down arrow", the drop opens but my current selection is not in view but is still the value I selected. If I start using arrow keys, I can't see where I am navigating.

I am doing the following in the video:
1. Mouse click into the input and selecting an option with my mouse.
2. Down arrow to open the suggestions again.
3. Down arrow to continue through selections (not in view since my original selection was outside of view).
4. Enter to select an option.

My expectation would be that upon using the down arrow after selecting an item, either:
a) My previously selected suggestion is in view when suggestions open
or
b) If I can't see my previous selection, once I press down arrow I should be navigating from the top of the suggestions list so I can see the active value.


https://user-images.githubusercontent.com/12522275/106532025-40d44780-64a4-11eb-84a2-a89ced64557c.mov